src/main/java/com/denizenscript/denizencore/objects/core/ListTag.java

```diff
@@ -1446,20 +1447,14 @@ else if (input.size() > 1) {
                     // For example: .get[1].to[3] on a list of "one|two|three|four" will return "one|two|three".
                     // Use "last" as the 'to' index to automatically get all of the list starting at the first index.
                     // For example: .get[3].to[last] on a list of "one|two|three|four" will return "three|four".
+                    // Negative indices can be used for the 'get' index as well as the 'to', counting from the end of the list.
+                    // For example: .get[-1].to[last] on a list of "one|two|three|four" will return "three|four".
                     // -->
                     if (attribute.startsWith("to", 2) && attribute.hasContext(2)) {
-                        int index2;
-                        if (CoreUtilities.equalsIgnoreCase(attribute.getContext(2), "last")) {
-                            index2 = object.size() - 1;
-                        }
-                        else {
-                            index2 = attribute.getIntContext(2) - 1;
-                        }
-                        if (index2 >= object.size()) {
-                            index2 = object.size() - 1;
-                        }
-                        if (index2 < 0) {
-                            index2 = 0;
+                        int index2 = CoreUtilities.parseIndex(object.objectForms, attribute.getContext(2));
+                        if (index2 < 0 || index2 < index) {

```

Comment:
The `index2 < index` here is new and thus adds an error state that previously was undocumentedly valid. This should not be done unless there's a specific reason for it

Reply:
Fair point, I'd added it since it would always return an empty list but should have asked first. I'll remove it